robomaker-sample-app-ci/src/aws-robomaker-sample-application-ci.ts

```diff
@@ -73,14 +73,19 @@ async function getSampleAppVersion() : Promise<string> {
   return Promise.resolve(version);
 }
 
-// If .rosinstall exists, run 'rosws update' and return a list of names of the packages that were added.
+// If .rosinstall exists, run 'vcs import' and return a list of names of the packages that were added in both workspaces.
 async function fetchRosinstallDependencies(): Promise<string[]> {
   let colconListAfter = {stdout: '', stderr: ''};
   let packages: string[] = [];
   // Download dependencies not in apt if .rosinstall exists
   try {
+    for (let workspace of ["robot_ws", "simulation_ws"]) {
```

Comment:
Let's put a very detailed comment here about how this serves the purpose of making the `generate-sources: true` option work properly.

Reply:
Good point.. added a description to why we are fetching in both the workspaces